src/e2e-test/features/gcs/source/GCSSourceToBigQuery.feature

```diff
@@ -285,3 +285,304 @@ Feature: GCS source - Verification of GCS to BQ successful data transfer
     Then Verify the pipeline status is "Succeeded"
     Then Get count of no of records transferred to target BigQuery Table
     Then Validate the values of records transferred from GCS bucket file is equal to the values of target BigQuery table
+
+    # New test scenarios added by Bijay on Nov 13
```

Comment:
Remove the comment added here

Reply:
Incorporated the review comments and pushed the latest code for your final review.